File: zgadula.cpp
//to jest program zgadula

#include <iostream>
#include <ctime>
#include <cstdlib>

using namespace std;


int szukana, zgadywana, licznik=0;

int main()

{

	srand (time (NULL));
	szukana = (rand () % 100); //wybiera pseudolosowo od 0 do 99

	cout << szukana << endl; 
	cout << "Zgadnij liczbę z zakresu od 0 do 99" << endl;


	do
	{	
	
		cin >> zgadywana;

	
	
		if (zgadywana>szukana)
		{
		cout << "Twoja liczba jest większa. Podaj mniejszą liczbę" << endl;
		licznik++;
		}		
	
		else if (zgadywana<szukana)
		{
		cout << "Twoja liczba jest mniejsza. Podaj większą liczbę" << endl;
		licznik++;		
		}

		

		
	} while(zgadywana!=szukana);
	
	
	cout << "Zgadłeś" << endl;
	cout << "Po " << licznik << " próbach" << endl;
	cout << "Gratulacje!!!";
	

return 0;

}

File: zgadula.py
import random
#a tak wyglądają komentarze
random.seed ()
while 1:
    a=int(input("Podaj a"))
    b=int(input("Podaj b"))
    if a>b:
        print("a>b")
    else:
           break
        
a=random.randint (0,100)
licznik=0
print ("Zgadnij liczbę od 0 do 100")
print (a)
while 1:
    
    liczba= int(input ())
    licznik+=1
    if liczba > a:
        print ("Podałeś większą liczbę, podaj mniejszą")
    elif liczba < a:
        print ("Podałeś mniejszą liczbę")
    else:
        print ("Zgadłeś za", licznik , "razem, ta liczba to: ", a)
        break
